swift-sdk/IterableNotificationMetadata.swift

```diff
@@ -8,68 +8,59 @@
 
 import Foundation
 
-/**
- `IterableNotificationMetadata` represents the metadata in an Iterable push notification
- */
-@objc public class IterableNotificationMetadata: NSObject {
-    @objc public var campaignId: NSNumber = NSNumber(value: 0)
-    @objc public var templateId: NSNumber? = nil
-    @objc public var messageId: String? = nil
+@objc public class IterableInAppMessageMetadata: IterableNotificationMetadata {
```

Comment:
Does this have to be a subclass of `IterableNotificationMetadata`? Looks like it's not setting the parent fields in the constructor.